presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -355,6 +361,12 @@ Property Name                             Description
 ``delete_mode``                            Optionally specifies the write delete mode of the Iceberg        ``merge-on-read``
                                            specification to use for new tables, either ``copy-on-write``
                                            or ``merge-on-read``.
+
+``metadata_previous_versions_max``         Optionally specifies the max number of old metadata files to     ``100``
```

Comment:
What would happen if metadata_delete_after_commit = false but metadata_previous_versions_max = 100? Would the files be deleted after 100? If not, shall we specify that metadata_previous_versions_max only works if metadata_delete_after_commit is set to true?

Reply:
`metadata_delete_after_commit = false` means that Iceberg wouldn't automatically delete the old and untracked metadata files after each commit. It wouldn't affect the functionality of `metadata_previous_versions_max`, which specify how many previous metadata files would be tracked in current metadata.

For example, if we set `metadata_previous_versions_max = 1` and `metadata_delete_after_commit = false` on the creation of an Iceberg table, and execute several insert operations which will lead in several metadata files.
We will get only 1 previous metadata entry in current metadata's log ( through `tableMetadata.previousFiles()`). But find that none of the metadata files were actually deleted.

On the other way, if we set `metadata_previous_versions_max = 1` and `metadata_delete_after_commit = true` and do the same things as above, we will find that only 2 metadata files remain.